spec/spec_helper.rb

```diff
@@ -1,13 +1,7 @@
 require 'capybara/rspec'
-require 'capybara/poltergeist'
-require 'webmock/rspec'
-require 'support/test_helpers'
-require 'percy'
```

Comment:
Should keep the `require 'percy'` line here — need to talk about the file paths to make the loading work correctly.

Reply:
Done.